src/applications/mhv-medical-records/tests/e2e/pages/NotesDetailsPage.js

```diff
@@ -2,36 +2,18 @@
 import BaseDetailsPage from './BaseDetailsPage';
 
 class NotesDetailsPage extends BaseDetailsPage {
-  /*
-    clickGotoNotesLink = (
-     /* Notes = defaultNotes,
-      waitForNotes = false,
-    ) => {
-      cy.intercept(
-        'GET',
-        '/my_health/v1/medical_records/notes',
-        Notes,
-      ).as('NotesList');
-      cy.get('[href="/my-health/medical-records/notes"]').click();
-      if (waitForNotes) {
-        cy.wait('@NotesList');
-      }
+  clickProgressNoteLink = (progressNoteHeading, index = 0) => {
+    // First make sure the heading is visible
+    cy.contains(progressNoteHeading, { includeShadowDom: true }).then(() => {
```

Comment:
Note: We search for the text first as this will make sure the page is loaded first.